src/components/screens/monitor/delegates/forgingDetails.css

```diff
@@ -0,0 +1,37 @@
+@import '../../../../app/variables.css';
+@import '../../../../app/mixins.css';
+
+.contentWrapper {
+  & h3 {
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 20px;
+    margin: 0;
+    position: relative;
+
+    &::after {
+      background-color: var(--color-platinum);
+      content: '';
+      height: 1px;
+      left: 0;
+      top: 25px;
+      position: absolute;
+      right: 95px;
+    }
+  }
+
+  & .contentBody,
+  & .totalForged {
+    margin-top: 15px;
+  }
+
+  & .totalForged {
+    @mixin contentNormal;
+  }
+
+  & .contentBody {
+    @mixin contentNormal bold;
+
+    color: var(--color-ultramarine-blue);
```

Comment:
I thought the content is blue because it's links to delegates, which already are blue and bold, so this wouldn't be needed.

Reply:
You're right. I replaced those with a `Link`. However I left the styling so that the `, ` wasn't included inside the link when hovering.